app/services/notify/notify-email.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+/**
+ * Sends an email using GOV.UK Notify
+ * @module NotifyEmailService
+ */
+
+const NotifyClient = require('notifications-node-client').NotifyClient
+
+const config = require('../../../config/notify.config.js')
+
+/**
+ * Send an email using GOV.UK Notify
+ *
+ * https://docs.notifications.service.gov.uk/node.html#send-an-email
```

Comment:
Email (and sms) can use the notify callback to let us know the status of the message. 

https://docs.notifications.service.gov.uk/node.html#set-up-callbacks

Do we want to use this and have a retry mechanism of our own.

Reply:
Will need to be handled on the legacy UI